Note about apsaradb (#49675)
metabase/metabase: docs/installation-and-operation/configuring-application-database.md

We recommend [PostgreSQL](#postgresql), but you can also use [MySQL](https://www.mysql.com/) or [MariaDB](https://www.mariadb.org/).

The minimum recommended version is MySQL 8.0.17 or MariaDB 10.2.2. The `utf8mb4` character set is required.

> NOTE: we don't support ApsaraDB MySQL, so please use ApsaraDB PostgreSQL

Reviewer: ```suggestion

We don't support ApsaraDB MySQL. You can instead use ApsaraDB PostgreSQL.
```
